src/Fmt_ast.ml

```diff
@@ -1443,17 +1443,19 @@ and fmt_expression c ?(box = true) ?pro ?epi ?eol ?parens ?(indent_wrap = 0)
       , [(Nolabel, l); (Nolabel, ({pexp_desc= Pexp_function cs} as r))] )
     when is_infix_id id && not c.conf.break_infix_before_func ->
       Cmts.relocate c.cmts ~src:pexp_loc ~before:loc ~after:loc ;
+      let xpc_rhs = sub_exp ~ctx r in
```

Comment:
Here the `xpc_rhs` copypasta name doesn't really make sense, `xr` is likely better.